src/plugins/geomap/demo/config-fr.js

```diff
@@ -93,7 +102,7 @@ var wet_boew_geomap = {
 						"oKS8uknovhu0LxRwllpRFyQDnTMta43C4fp0qb9rxVlTXexNY9S1Lb5rhp4Ee07LSpkMiQhNYrRCDzZJks0XKEyI45Ha7RaScGT9YRXFHd5nNEH8C+eGD9m6tNTgAAAAASUVORK5CYII=",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de charbon"
+						name: "Coal mines"
```

Comment:
Do we need to use the English name? Can we keep the French name?

Reply:
Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448